<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TableWindow</class>
 <widget class="QMainWindow" name="TableWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>850</width>
    <height>651</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="filtersLayout">
        <property name="sizeConstraint">
         <enum>QLayout::SetMinAndMaxSize</enum>
        </property>
       </layout>
      </item>
      <item>
       <widget class="QTableView" name="tableView">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLineEdit" name="id">
          <property name="placeholderText">
           <string>id</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton">
          <property name="text">
           <string>-</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLineEdit" name="name">
          <property name="placeholderText">
           <string>ФИО</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="course">
          <property name="placeholderText">
           <string>Курс</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="group">
          <property name="placeholderText">
           <string>Группа</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="photoPath">
          <property name="text">
           <string/>
          </property>
          <property name="placeholderText">
           <string>Выберите фото</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="addPhoto">
          <property name="text">
           <string>...</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDateEdit" name="date"/>
        </item>
        <item>
         <widget class="QPushButton" name="addStudentButton">
          <property name="text">
           <string>+</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>850</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
